packages/react-dom-bindings/src/client/ReactFiberConfigDOM.js

```diff
@@ -2076,6 +2077,222 @@ export function subscribeToGestureDirection(
   }
 }
 
+type EventListenerOptionsOrUseCapture =
+  | boolean
+  | {
+      capture?: boolean,
+      once?: boolean,
+      passive?: boolean,
+      signal?: AbortSignal,
+      ...
+    };
+
+type StoredEventListener = {
+  type: string,
+  listener: EventListener,
+  optionsOrUseCapture: void | EventListenerOptionsOrUseCapture,
+};
+
+export type FragmentInstance = {
+  _fragmentFiber: Fiber,
+  _eventListeners: void | Array<StoredEventListener>,
```

Comment:
This should always be initialized. Otherwise the hidden class will deopt when you add this property and all callsite will have to recompile which is really bad. Always initialize all fields in the constructor.

Also, even when you initialize, try to use `null` instead of `undefined` because `undefined` is sometimes treated as missing in terms of hidden classes in V8.

So this should be `null | Array<StoredEventListener>` and the constructor needs `this._eventListeners = null`.